languages/cpp/generic/cpp_to_generic.ml

```diff
@@ -2103,8 +2192,9 @@ let map_any env x : G.any =
 (*****************************************************************************)
 (* Entry point *)
 (*****************************************************************************)
-let any x =
+let any parsing_opt x =
```

Comment:
maybe you can use a default parameter, like `let any ?(cpp_parsing_opt=None) x = ...
so we don't have those ugly  calls like 'any None xxx' which are not super readable.